ch1-arraysAndStrings/wmassey/README.md

```diff
@@ -0,0 +1,72 @@
+# Cracking the Coding Interview
+##  🔔 Background Info 
+
+My Coding solutions for _Cracking the Coding Interview_, chapter 01: _Arrays and Strings_
+
+---
+## Technology Framework 
+🚀 This project was built with  
+
+
+![made-with-VS-Code](https://img.shields.io/badge/1.60.0-blue?&labelColor=black&label=VS%20COde&logo=visualstudiocode&logoColor=white)
+
+![made-with-Python](https://img.shields.io/badge/3.9.6-blue?&labelColor=black&label=Python&logo=python&logoColor=white)
+
+---
+
+
+## Requirements
+✔️ This project requires the following installations
+
+![made-with-Python](https://img.shields.io/badge/3.9.6-blue?&labelColor=black&label=Python&logo=python&logoColor=white)
+
+---
+## Content
+
+|Problem #| Problem Title        | Stats |
+|---------|----------------------|-------|
+| 1-1     |is_unique             |  ✅  |
+| 1-2     |check_permutation     |  ✅  |
+| 1-3     |URLify                |  ✅  |
+| 1-4     |palindrome_permutation|  🤬  |
+| 1-5     |one_way               |  ❌  |
+| 1-6     |string_compression    |  ✅  |
+| 1-7     |rotate_matrix         |  ❌  |
+| 1-8     |zero_matrix           |  ❌  |
+| 1-9     |string_rotation       |  ✅  |
+
+
+
+---
+## 🤝 Contribute
+ 
+This project is Open Source. Feedback, Questions and [**Pull Requests**](https://github.com/seraph776/REPONAME/pulls) are welcome. 
```

Comment:
I'm not sure it makes sense to include this section in this README. Can you remove this section?

Reply:
I apologize for the long delay in responding to this! I am still new to `Git` and the `Git work` flow, and I'm just now realizing you requested these changes. Please let me know if everything looks good! Thanks again for your patience!